cmd/simpleargs/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+	"github.com/google/go-github/v26/github"
+	"gopkg.in/alecthomas/kingpin.v2"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"regexp"
+)
+
+var regex string
+var eventfile string
+var writepath string
+
+func writeArgs(groups []string) {
+	for i, group := range groups[1:] {
+		data := []byte(group)
+		filename := fmt.Sprintf("ARG%d", i)
```

Comment:
maybe to make it a bit more readable
```suggestion
		filename := fmt.Sprintf("ARG_%d", i)
```